internal/mode/static/nginx/config/upstreams.go

```diff
@@ -29,9 +33,29 @@ const (
 	plusZoneSizeStream = "1m"
 )
 
-func (g GeneratorImpl) executeUpstreams(conf dataplane.Configuration) []executeResult {
-	upstreams := g.createUpstreams(conf.Upstreams)
+// UpstreamMap holds a map which maps upstream name to http.Upstream.
+type UpstreamMap struct {
+	nameToUpstream map[string]http.Upstream
```

Comment:
I'd rather this be named `upstreams` and a comment can describe what the key and value are (name and upstream)